layouts/shortcodes/code-example.html

```diff
@@ -1,51 +1,53 @@
 {{- $pathURL := (print "/content" (.Get "pathURL"))}}
 {{- $files := readDir $pathURL -}}
-<div class="example-tabs">
-    {{ $.Scratch.Set "tabClass" "active" }}
+<div class="code-example">
+    <div class="example-tabs">
+        {{ $.Scratch.Set "tabClass" "active" }}
+        {{- range $files }}
+            {{ $filename_arr := split .Name "." }}
+            {{ $filename := index $filename_arr 0}}
+            {{ $filename_ext := index $filename_arr 1 }}
+
+            {{ if eq $filename ($.Get "exampleId") }}
+                <button class="example-tab {{ $.Scratch.Get "tabClass"}} {{$.Get "example-id"}}" id="{{.Name}}">
```

Comment:
The clickable tab doesn't change mouse cursor to pointer. Minor.

Reply:
Fixed in custom.css.